client/my-sites/sidebar/style.scss

```diff
@@ -0,0 +1,25 @@
+.sidebar__sparkline {
+	width: 80px;
+	height: 24px;
+	position: absolute;
+	right: 16px;
+	top: 8px;
+
+	@include breakpoint( "<660px" ) {
+		top: 13px;
+		right: 20px;
+	}
+}
+
+.sidebar__menu li.stats.selected .sidebar__sparkline {
+	display: none;
+
+	@include breakpoint( "<480px" ) {
```

Comment:
Ah I didn't see it at 660 myself but was able to re-create at the lower breakpoint.  I will update to 660.